src/lib/viewers/archive/ArchiveLoader.js

```diff
@@ -0,0 +1,42 @@
+import AssetLoader from '../AssetLoader';
+import ArchiveViewer from './ArchiveViewer';
+import { ORIGINAL_REP_NAME } from '../../constants';
+
+// Order of the viewers matters. Prefer original before others. Go from specific to general.
+const VIEWERS = [
+    {
+        NAME: 'Archive',
+        CONSTRUCTOR: ArchiveViewer,
+        REP: 'json',
+        EXT: ['zip'],
+    },
+    {
+        NAME: 'Archive',
+        CONSTRUCTOR: ArchiveViewer,
+        REP: ORIGINAL_REP_NAME,
```

Comment:
Is this case possible? Is the ArchiveViewer equipped to handle it?

Reply:
This is only for testing purpose, because currently the file api only gives us ORIGINAL_REP_NAME. Will remove it after the real api is in prod.

Reply:
> Is this case possible? Is the ArchiveViewer equipped to handle it?

This is only for testing purpose, because currently the file api only gives us ORIGINAL_REP_NAME. Will remove it after the real api is in prod.